### DISCUSSION TITLE
While practicing the ansible module getting the password related error.

### DISCUSSION THREAD

**Discussion by NabiDasta** [learndevopsonline/learndevopsonline#1585] [Q&A] [answered]
Hi,

while practice the ansible module getting below error. I am executive the ansible backend-dev  command there i am getting password related error. even i gave the right password also it's getting error.

My public IP 's: 
![image](https://github.com/learndevopsonline/learndevopsonline/assets/148522896/07a0759c-72f6-4b57-afd5-46cd8490b3e5)

![image](https://github.com/learndevopsonline/learndevopsonline/assets/148522896/094a9e3d-1883-4be2-b58a-a4ae687a6a47)

**Comment by sreedharm07** [comment]
Hi @NabiDasta 
your workstation ip was cropped in pic please share you public ip

**Reply by sreedharm07** [reply]
@NabiDasta 
can you please ping your workstation public ip

**Reply by NabiDasta** [reply]
Sorry please find My workstation IP : 35.153.166.107

Thanks,
Nabisaheb Vannurappagari

**Reply by sreedharm07** [reply, marked as answer]
Hi @NabiDasta 
before loading schema you **need to download mysql**  in your backend.yml file i dont see any step to download mysql, you missed that step
please add it and try it will run without error

**Reply by NabiDasta** [reply]
Got it Thank you sreedhar.
